neopia/neosoco.py

```diff
@@ -518,4 +534,4 @@ def buzzer_by_port(self, port='in1'):
             value = self._convert_input_port_scale(port, 65)
             self.write(Neosoco.NOTE, value)
         else:
-            raise TypeError
```

Comment:
If I were a strict reviewer, I'll request to remove the meaningless code change in the commit and make re-PR, but I'll not request until re-PR. To avoid this mistake, you can check one more if there is a meaningless code change  before making a real commit.